stable/distribution/templates/distribution-statefulset.yaml

```diff
@@ -16,10 +19,8 @@ spec:
   selector:
     matchLabels:
       app: {{ template "distribution.name" . }}
-      chart: {{ template "distribution.chart" . }}
       release: {{ .Release.Name }}
       role: {{ template "distribution.name" . }}
-      component: {{ .Values.distribution.name }}
```

Comment:
Same here

Reply:
please refer https://github.com/jfrog/charts/pull/920/files#diff-c1b95a0c0cd60d8f73ba630822cc6ae1L445-L449 - This will be a breaking change for statefulset upgrades.

Reply:
If the removed labels are a breaking change I suggest you don't remove them @chukka